python-connectors/kayrros_retrieve-dataset/connector.json

```diff
@@ -0,0 +1,60 @@
+/* This file is the descriptor for the Custom python dataset kayrros_retrieve-dataset */
```

Comment:
The comments can be removed from this file

Reply:
solved: [commit](https://github.com/dataiku/dss-plugin-kayrros/commit/64f26a2e5f929bd0b3ac86e17610160bce6e4ab3)